maquetado/index.html

```diff
@@ -98,5 +98,12 @@ <h3 class="tech-year">Release: 2005</h3>
       </div>
     </div>
   </div>
+  <div class="footer">
+    <span class="footer-paragraph">WE ARE GAME CHANGERS</span>
+    <img src="./assets/wolox_logo_white.svg" alt="wolox logo white">
+    <div class="copyright-container">
```

Comment:
I don't think this `div` is necessary here. What about using flex to align and justify all the items within the `footer` element?